Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,32 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+
+const HeaderTitle = () => {
+  //   let month = new Array();
+  //   month[0] = "Jan";
+  //   month[1] = "Feb";
+  //   month[2] = "Mar";
+  //   month[3] = "Apr";
+  //   month[4] = "May";
+  //   month[5] = "Jun";
+  //   month[6] = "July";
+  //   month[7] = "Aug";
+  //   month[8] = "Sept";
+  //   month[9] = "Oct";
+  //   month[10] = "Nov";
+  //   month[11] = "Dec";
+
+  //   let d = new Date();
+  //   let n = month[d.getMonth()];
+  //   document.getElementById("demo").innerHTML = n;
+  return (
+    <React.Fragment>
+      <h1>Lambda School</h1>
+      <span>@LambdaSchool - {moment().format("D MMM")}</span>
```

Comment:
To use the moment library, you have to first install moment as a dependency with `yarn add moment` in your terminal and then import it into this component.